assets/data/cards_pt-br.json

```
@@ -482,6 +476,21 @@
         "Conceito",
         "Back-end"
```

Aqui pode deixar somente a tag de `Back-end`.

```suggestion
        "Back-end"
```